src/_experimental.scss

```diff
@@ -159,3 +169,102 @@
 		}
 	}
 }
+
+
+.provisional {
+	.alert-info > :first-child::before,
```

Comment:
I think this block is to undo the old style. Can you add a comment before that said that?